docs/components/servo/_index.md

```diff
@@ -6,18 +6,20 @@ type: "docs"
 description: "A hobby servo is a special type of small motor whose position you can precisely control."
 tags: ["servo", "components"]
 icon: "/components/img/components/servo.svg"
+no_list: true
 # SME: #team-bucket
 ---
 
-The Viam *servo* component supports hobby servos: small motors with built-in closed-loop control.
-Servos can be useful in robotics because their position is easily and precisely controlled.
+A *servo* is a ["RC" or "hobby" servo motor](https://learn.adafruit.com/adafruit-motor-selection-guide/rc-servos).
```

Comment:
Either
```suggestion
The servo component supports ["RC" or "hobby" servo motors](https://learn.adafruit.com/adafruit-motor-selection-guide/rc-servos).
```
or
```suggestion
A *servo* is an ["RC" or "hobby" servo motor](https://learn.adafruit.com/adafruit-motor-selection-guide/rc-servos).
```
I think the former sounds less tautological.

Reply:
I like the first one